### DISCUSSION TITLE
WaterfallPanel 请问支持ItemSourc 绑定吗？

### DISCUSSION THREAD

**Discussion by heartacker** [HandyOrg/HandyControl#1018] [Q&A] [answered]
该如何做呢？谢谢高手回答

**Comment by Yopler** [comment, marked as answer]
I don't read chinese, I suppose it is corresponding as an issue that you open at the same time. 
[Here my answer ](https://github.com/HandyOrg/HandyControl/issues/1019#issuecomment-945618205) : 
> There is no `ItemSource` Property in WaterfallPanel.
> 
> WaterFallPanel inherited from Panel (see [WaterFallPanel.cs](https://github.com/HandyOrg/HandyControl/blob/master/src/Shared/HandyControl_Shared/Controls/Panel/WaterfallPanel.cs)). See it as a `StackPanel` or a `WrapPanel`.
> 
> If you really want to use an `ItemSource` property, you can try this idea :
> 
> ```
> <ListBox ItemSource="{Binding YourBind}">
>      <ListBox.ItemsPanel>
>            <ItemPanelTemplate>
>                  <hc:WaterFallPanel Groups="2"/>
>           </ItemPanelTemplate>
>     </ListBox.ItemsPanel>
>     <ListBox.ItemTemplate>
>          <DataTemplate>
>             ...
>          </DataTemplate>
>    </ListBox.ItemTemplate>
> </ListBox>
> ```
> 
> I never try this with the `WaterFallPanel`, but with `StackPanel `it is working. You should give it a try 😄

**Reply by heartacker** [reply]
谢谢，有效。感谢

**Reply by heartacker** [reply]
It works，Thanks
